gamemodes/terrortown/gamemode/shared/lang/fr.lua

```diff
@@ -1103,12 +1103,12 @@ L.submenu_guide_gameplay_title = "Jouabilité"
 L.submenu_guide_roles_title = "Rôles"
 L.submenu_guide_equipment_title = "Équipement"
 
-L.submenu_bindings_bindings_title = "Bindings"
+L.submenu_bindings_bindings_title = "Commandes"
```

Comment:
Further down you translated it with "touches"

Reply:
fixed